README.md

```diff
@@ -22,7 +22,7 @@ Notes:
 * On slower systems, the first time the CODE Server is started there may be a noticeable delay (also applies on subsequent runs if `/tmp` whenever cleared)
 
 ## System requirements
-- Linux x86-64 or ARM64 (aarch64) platform
+- Linux x86-64 (ARM64/aarch64 platform is not supported for all users at this time, see [Issue 226](https://github.com/CollaboraOnline/richdocumentscode/issues/226))
```

Comment:
This is perfect, what a brilliant change. We need this merged so people don't waste their time!